vllm/executor/multiproc_gpu_executor.py

```diff
@@ -37,6 +37,11 @@ def _init_executor(self) -> None:
         # Disable torch async compiling which won't work with daemonic processes
         os.environ["TORCHINDUCTOR_COMPILE_THREADS"] = "1"
 
+        # Set OMP_NUM_THREADS to 1 if it is not set explicitly, avoids CPU
+        # contention amongst the shards
+        if os.environ.get("OMP_NUM_THREADS") is None:
```

Comment:
nit: maybe slightly cleaner
```suggestion
        if "OMP_NUM_THREADS" is not in os.environ:
```

Reply:
@tjohnson31415 oops sorry for the extra `is`